app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class FriendshipsController < ApplicationController
+  before_action :set_user
+
+  def create
+    friendship = current_user.active_friendships.new(followed: @user)
+    if friendship.save
+      redirect_to @user, notice: t('.notice')
+    else
+      redirect_to @user, alert: t('.alert')
+    end
+  end
+
+  def destroy
+    friendship = current_user.active_friendships.find_by(params[:id])
+    if friendship
+      friendship.destroy
```

Comment:
safe navigation operator を使うとよいと思います。

```suggestion
    if friendship&.destroy
```